DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -474,5 +481,84 @@ public void Get_section_content_should_have_default_course_settings_when_json_is
             result.Should().NotBeNull();
             result!.CourseSettings.Should().BeEquivalentTo(defaultSettings);
         }
+
+        [Test]
+        public void Get_section_content_next_section_id_should_return_id_if_at_start_of_list()
+        {
+            // Given
+            const int customisationId = 15853;
+            const int candidateId = 1;
+            const int sectionId = 382;
+            const int expectedNextSectionId = 383;
+            // When
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            //Then
+            result.NextSectionId.Should().Be(expectedNextSectionId);
+        }
+
+        [Test]
+        public void Get_section_content_next_section_id_should_return_id_if_in_middle_of_list()
+        {
+            // Given
+            const int customisationId = 15853;
+            const int candidateId = 1;
+            const int sectionId = 383;
+            const int expectedNextSectionId = 384;
+            // When
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            //Then
+            result.NextSectionId.Should().Be(expectedNextSectionId);
+        }
+
+        [Test]
+        public void Get_section_content_next_section_id_should_return_null_if_at_end_of_list()
+        {
+            // Given
+            const int customisationId = 15853;
+            const int candidateId = 1;
+            const int sectionId = 386;
+            // When
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            //Then
+            result.NextSectionId.Should().BeNull();
+        }
+
+        [Test]
+        public void Get_section_content_next_section_id_should_skip_empty_section()
+        {
+            // Given
+            const int customisationId = 18366;
+            const int candidateId = 210934;
+            const int sectionId = 974;
+
+            // The next section ID in this Application is 975, but the next section with a tutorial selected in
+            // CustomisationTutorials is 978
+            const int expectedNextSectionId = 978;
```

Comment:
What about if the next section only has a diagnostic or post learning assessment? 🙂